UI rework

RenatKunafin opened the pull request:
1. Splitted up front end on three files - added css and js files
2. UI - reworked. Place cursor to right screen border for show up the menu, or use shortcuts (see readme)
3. Dependencies optimized - excluded jQuery, Popper.js, Bootstrap JS 
4. Some code style improvements according PEP8 standard

iamtrump commented:
Dear Renat! Thank you for the contribution, you did a great job, but unfortunately I can’t merge it into master branch because there are few issues:

- Video doesn’t play.

- I can’t enter the menu from my mobile device (iPhone SE).

As soon as you fix these issues, I'll merge it with a pleasure.

RenatKunafin commented:
1. Fixed bug with video playing
2. Added mobile compatibility. Just long tap on right side of screen for opening menu, and short tap on left side fo closing.
3. Added full search. If you need some channel from full list - mark a checkbox below search input.

iamtrump commented:
Thanks for contribution!
